src/build/parse.rs

```diff
@@ -203,6 +203,29 @@ pub fn generate_asts(
                     namespaces::compile_mlmap(package, module_name, &build_state.bsc_path);
                     let mlmap_hash_after = helpers::compute_file_hash(&compile_path);
 
+                    let suffix = package
+                        .namespace
+                        .to_suffix()
+                        .expect("namespace should be set for mlmap module");
+                    // copy the mlmap to the bs build path for editor tooling
+                    let base_build_path = package.get_build_path() + "/" + &suffix;
+                    let base_bs_build_path = package.get_bs_build_path() + "/" + &suffix;
+                    let _ = std::fs::copy(
```

Comment:
I think it might be time to introduce something like `anyhow` or another way to do nice error handling - more of a note for the future

Reply:
Ah cool anyhow looks nice. This stuff also reminds me to refactor the string based paths to proper paths :)